tests/integration/pages/desktop/home.page.js

```diff
@@ -0,0 +1,31 @@
+/* global $ */
+
+"use strict";
+
+class HomePage {
+  /* Represents the Home page */
+
+  waitForPageToLoad() {
+    this.monitorLogo.waitForExist(5000);
+    return this;
+  }
+
+  get monitorLogo() { return $(".fx-monitor-logotype"); }
+  get breachCard() { return new BreachCard(); }
+  get breachEmailAddress() { return $("#scan-email"); }
+  get checkBreachesButton() {
+    $(".input-group-button > input:nth-child(1)").click();
```

Comment:
n-th child!
Not sure if we want to put a more convenient class/handle on that submit button to make this any easier on you.
Or not sure if we can abuse the system with something like:

```js
$(".input-group-button > input[type='submit']).click(); // if this even works
```

I'd be reluctant to bend it too far and try reading the button's label, since that will change with different locales (ie: `input[value='Check for Breaches']` feels a bit risky risky).

Reply:
Classes on everything would make it easier but I know that is boring so..this works, and if it gets updated it's an easy change